src/app/tracing/util/labels.ts

```diff
@@ -0,0 +1,43 @@
+type Labels = {
+    [key:string]: string, 
+};
+
+
+export const LABELS:Labels = {
+    id: 'ID',
+    lot: 'Lot',
+    lat: 'Latitude',
+    lon: 'Longitude',
+    dateIn: 'Delivery Date Arrival',
+    dateOut: 'Delivery Date',
+    source: 'Source',
+    target: 'Target',
+    originalSource: 'Original Source',
+    originalTarget: 'Original Target',
+    incoming: 'Incoming',
+    outgoing: 'Outgoing',
+    contains: 'Contains',
+    isMeta: 'Is Meta Station',
+    contained: 'Is Meta Member',
+    forward: 'Forward Trace',
+    backward: 'Backward Trace',
+    observed: 'Trace Type',
+    outbreak: 'Outbreak',
+    crossContamination: 'Cross Contamination',
+    killContamination: 'Kill Contamination',
+    commonLink: 'Common Link',
+    score: 'Score',
+    weight: 'Weight',
+    anonymisationLabel: 'Anonymisation Label',
+    anonymizedName: 'Anonymized Name',
+    name: 'Name',
+    address: 'Address',
+    country: 'Country',
+    typeOfBusiness: 'Type of Business',
+    selected: 'Selected',
+    invisible: 'Invisible',
+    product: 'Product',
+    amount: 'Amount',
+    stationLabel: 'StationLabel',
+    deliveryLabel: 'DeliveryLabel',
```

Comment:
There was a space missing in the orginial string. I didn't change that, because I wasn't sure, if there is a reason for that. Is this intentional?

Reply:
The which space are you referring?

Reply:
@ruegenbfr There are a couple more spots, where I wasn't sure, if I should change the namings as well, or if that would break something:

- /Users/hanna/projects/fcl-client/src/app/tracing/io/ext-data-constants.v1.ts 
- /Users/hanna/projects/fcl-client/src/app/tracing/io/data-importer/shared.ts --> `function createDefaultDeliveryHRules()` --> `id: DELIVERY_DEFAULT_HIGHLIGHTING_RULE_ID_PREFIX + 'Outbreak'`, etc
- the JSON files for the sample data